src/environment.js

```diff
@@ -19,7 +19,7 @@ const ENV_VARS = {
     return process.env.PORTIS_DAPP_ID || ''
   },
   SENTRY_DSN() {
-    const dsn = process.env.SENTRY_DSN || ''
+    const dsn = process.env.NEXT_PUBLIC_SENTRY_DSN || ''
```

Comment:
Right, let's actually rollback to explicitly providing the environment variable in the config and avoid using the Vercel frontend for now to do this!

Fleek won't have access to these variables.